hs_core/templates/pages/create-resource.html

```diff
@@ -196,68 +196,66 @@ <h3>Drop files here or click to upload</h3>
                                 </div>
                             </div>
 
-                            <div class="row">
-                                <div class="col-xs-12">
-                                    <hr>
-                                    <label>To load files from an iRODS server including your iRODS user HydroShare server:</label>
-
-                                    <div id="irods-browse">
-                                        <div id="log-into-irods">
-                                            <a id="btn-signin-irods" type="button" class="btn btn-info"
-                                               data-toggle="modal"
-                                               data-target="#irodsSignin"><span
-                                                    class="glyphicon glyphicon-log-in"></span>&nbsp;&nbsp;Log in to iRODS...</a>
-                                        </div>
-                                        <br>
-                                        <input type="hidden" id="irods-username" name="irods-username" value="">
-                                        <input type="hidden" id="irods-password" name="irods-password" value="">
-                                        <input type="hidden" id="irods-host" name="irods-host" value="">
-                                        <input type="hidden" id="irods-port" name="irods-port" value="">
-                                        <input type="hidden" id="irods-zone" name="irods-zone" value="">
-                                        <input type="hidden" id="irods_file_names" name="irods_file_names" value="">
-                                        <input type="hidden" id="irods_federated" name="irods_federated" value="">
-
-                                        <a id="btn-select-irods-file" type="button"
-                                           class="btn btn-success hidden"
-                                           data-toggle="modal" data-target="#irodsContent">
-                                            Browse iRODS...
-                                        </a>
+                            <div>
+                                <hr>
+                                <label>To load files from an iRODS server including your iRODS user HydroShare server:</label>
+
+                                <div id="irods-browse">
+                                    <div id="log-into-irods">
+                                        <a id="btn-signin-irods" type="button" class="btn btn-info"
+                                           data-toggle="modal"
+                                           data-target="#irodsSignin"><span
+                                                class="glyphicon glyphicon-log-in"></span>&nbsp;&nbsp;Log in to iRODS...</a>
+                                    </div>
+                                    <br>
+                                    <input type="hidden" id="irods-username" name="irods-username" value="">

```

Comment:
I like your articulate the id for each tag. This is a tremendous help for automation testing.